syntax = "proto3";
package accesslogic;

message HelloReq{
	bytes key = 1;
}

message HelloResp{
	bytes value = 1;
}

message FileSegmentInfo{
    bytes filename              = 1;
    repeated bytes segmentfile  = 2;
    repeated bytes segmentorder = 3;
} 

service AccessLogicService {
	rpc Hello (HelloReq) returns (HelloResp);
}
